test/src/main/java/com/valqueries/SqlServerDataSourceProvider.java

```diff
@@ -18,7 +18,7 @@ private SqlServerDataSourceProvider() {
 		HikariConfig config = new HikariConfig();
 
 
-		config.setJdbcUrl(System.getProperty("db.url", "jdbc:sqlserver://localhost;databaseName=valqueries"));
```

Comment:
how is the DB-name specified with MSSQL then?

Reply:
It uses the default mssql db, which works fine for these tests. I would like to find a way of creating the database schema upon boot of the docker instance, but i haven't figured out how to do that yet.